packages/manager/modules/hub/src/components/catalog-items/controller.js

```diff
@@ -1,4 +1,18 @@
 export default class {
+  constructor() {
+    this.loading = true;
+  }
+
+  $onInit() {
+    this.items
```

Comment:
i expect items to be a list of items, maybe use a method like fetchItems instead ? to make it clear that it's a promise